### DISCUSSION TITLE
Can the GridView handle large amounts of records?

### DISCUSSION THREAD

**Discussion by donnyv** [unoplatform/uno#5797] [Q&A]
I'm new to UNO and I'm researching to see if UNO can be used with a new project I'm working on. Can the GridView handle large amounts of records like 100k+? Also does it use virtual scrolling? Can it handle infinite scrolling?

**Comment by davidjohnoliver** [comment]
Can you tell us which platform or platforms you're interested in targeting?

**Reply by donnyv** [reply]
Windows, Mac and Linux
